pom.xml

```diff
@@ -990,12 +1015,48 @@
                                 <version>${mtf-tools.version}</version>
                                 <classifier>mule-plugin</classifier>
                             </dependency>
+                            <dependency>
+                                <groupId>org.apache.commons</groupId>
+                                <artifactId>commons-dbcp2</artifactId>
+                                <version>2.9.0</version>
```

Comment:
Can we please move this version number to a property?

Reply:
Done